oshc/main/templates/contest_submission.html

```diff
@@ -4,8 +4,8 @@
 
 <div class="jumbotron text-xs-center">
   <div class="container text-center">
-      <h1 class="display-3">Thank You!</h1>
-      <p class="lead"><strong>Your contest has been submitted for review.</strong></p>
+      <h1 class="display-3" style="text-align: center">Thank you!</h1>
+      <p class="lead" style="text-align: center"><strong>Your contest has been submitted for review.</strong></p>
```

Comment:
This isn't working for me, can you please share screenshots?

Reply:
![contest submission screenshot](https://user-images.githubusercontent.com/30446949/31833407-5e3b118a-b5c2-11e7-844f-c26608799d8c.PNG)

Reply:
center of the page, like vertically and horizontally in the center of the page. I will update the description.